modules/mapreduce/src/main/java/org/apache/fluo/mapreduce/FluoOutputFormat.java

```diff
@@ -43,8 +43,7 @@
   private static String PROPS_CONF_KEY = FluoOutputFormat.class.getName() + ".props";
 
   @Override
-  public void checkOutputSpecs(JobContext arg0) throws IOException, InterruptedException {
-  }
+  public void checkOutputSpecs(JobContext arg0) throws IOException, InterruptedException {}
```

Comment:
This kept getting auto formatted by maven.

Reply:
Maybe there was a previous commit where this was not properly formatted.